draft-ietf-tls-esni.md

```diff
@@ -153,21 +155,23 @@ and the provider's server relays the connection back to the
 backend server, which is the true origin server. The provider does
 not have access to the plaintext of the connection.
 
-## SNI Encryption
+## ClientHello Encryption
 
-SNI encryption requires that each provider publish a public key and
-metadata which is used for SNI encryption for all the domains for
+SNI encryption works by encrypting the entire ClientHello, including
+the SNI and any additional extensions such as ALPN.
+This requires that each provider publish a public key and
+metadata which is used for ClientHello encryption for all the domains for
 which it serves directly or indirectly (via Split Mode). This document
 defines the format of the SNI encryption public key and metadata,
-referred to as an ESNI configuration, and delegates DNS publication
+referred to as an ECHO configuration, and delegates DNS publication
 details to {{!HTTPSSVC=I-D.nygren-dnsop-svcb-httpssvc}}, though other
 delivery mechanisms are possible. In particular, if some of the
-clients of a private server are applications rather than Web browsers,
+clients of a private server are applications rather than web browsers,
 those applications might have the public key and metadata
 preconfigured.
 
-When a client wants to form a TLS connection to any of the domains
-served by an ESNI-supporting provider, it constructs a ClientHello in
+When a client wants to create a TLS connection to any of the domains
```

Comment:
What's wrong with "form"?

Reply:
Nothing in particular. We can swap it back.